ibis/backends/flink/tests/test_ddl.py

```diff
@@ -552,3 +553,48 @@ def test_read_json(con, data_dir, tmp_path, table_name):
 
     con.drop_table(table_name)
     assert table_name not in con.list_tables()
+
+
+@pytest.fixture(scope="module")
+def functional_alltypes(con):
+    return con.table("functional_alltypes")
+
+
+@pytest.mark.parametrize(
+    "table_name",
+    [
+        "astronauts",
+        "awards_players",
+        "diamonds",
+        "functional_alltypes",
+    ],
+)
+def test_to_csv(con, functional_alltypes, tmp_path, table_name):
+    table = con.table(table_name)
+    out_path = tmp_path / "out.csv"
+    con.to_csv(table, out_path)
+
+    source_df = table.to_pandas()
+    out_df = pd.read_csv(out_path)
+
+    assert source_df.shape == out_df.shape
+
+
+@pytest.mark.parametrize(
+    "table_name",
+    [
+        "astronauts",
+        "awards_players",
+        "diamonds",
+        "functional_alltypes",
+    ],
+)
+def test_to_parquet(con, tmp_path, table_name):
```

Comment:
Any reason we need these new tests as opposed to depending on the ones that run as part of `ibis/backends/tests/test_export.py`?

Reply:
There is not a particular reason. They are simply the outcome of the need I had for having "basic tests in controlled environment" while developing the functionality. An extra benefit I got from them was that I learned Flink does not take the header row in CSV files into account while creating table based on them, for a good reason :)

Do you think I should delete them?

Reply:
We can leave them for now.